src/baseIcons/icon-radio-button-on.svg

```diff
@@ -1,4 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<svg width="30px" height="30px" viewBox="0 0 30 30" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
-    <circle cx="15" cy="15" r="8" />
+<svg  viewBox="0 0 10 10" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <title>Icons/Basic icons/radio-button-on</title>
+    <g  id="Icons/Basic-icons/radio-button-on" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
+        <circle  id="Oval-6" fill="#2A6EBB" cx="5" cy="5" r="5"></circle>
+    </g>
```

Comment:
Default color seems to be now blue ( #2A6EBB ) instead of same "black" ( #292929 ) as others.